test/values.ts

```diff
@@ -422,6 +423,13 @@ export const mockWallet: UnlockableWallet = {
   computeSharedSecret: jest.fn(),
 }
 
+export const mockViemWallet: Partial<ViemWallet> = {
+  prepareTransactionRequest: jest.fn(),
+  signTransaction: jest.fn(),
+  signTypedData: jest.fn(),
+  signMessage: jest.fn(),
+}
```

Comment:
Looks like we should consolidate this with `__mocks/src/web3__` to avoid confusion?
i.e. having one default mockViemWallet, no?